test_stack.py

```diff
@@ -0,0 +1,52 @@
+from stack import StackItem
+from stack import StackFrame
+import pytest
+
+
+def test_item_data():
+    # Tests that "Bacon" is returned when calling .data on item
+    bacon = StackItem("Bacon")
+    assert bacon.data == "Bacon"
+
+
+def test_create_stack_frame():
+    # Tests that an empty stack can be created
+    new_stack = StackFrame()
+    assert new_stack
+
+
+def test_stack_push():
+    # Tests that "Bacon" is first item when pushed to stack
+    bacon = StackItem("Bacon")
+    new_stack = StackFrame()
+    new_stack.push(bacon)
```

Comment:
See how we need to interact with your StackItem class to use the Stack?  That's not good.  It should be an internal implementation detail.  I should be able to push a string onto the stack, or a number, or an object, and get that thing back when I pop the stack.